resources/views/content/scripts.twig

```diff
@@ -56,21 +51,23 @@
 <script type="x/template" data-component="item-search">
     {{ component( "Findologic::ItemList.Components.ItemSearch") }}
 
-    <form method="GET" action="/search" onsubmit="return flSearchSubmitEventHandler(event, this);">
+    <form method="GET" action="/search" @submit="search()">
         <div class="container-max">
             <div class="position-relative">
                 <div class="d-flex flex-grow-1 position-relative my-2">
                     <input name="query"
                            type="search"
                            class="search-input flex-grow-1 px-3 py-2"
                            ref="searchInput"
-                           @keyup.enter="prepareSearch()"
+                           {% if minSearchLength %}
```

Comment:
Is it possible that this does not exist?

Reply:
Maybe you want to change this to `> 0`, so it is clear that this will always be a number?